addOns/pscanrulesAlpha/src/test/java/org/zaproxy/zap/extension/pscanrulesAlpha/JSFunctionPassiveScannerUnitTest.java

```diff
@@ -185,7 +186,9 @@ public void shouldAlertOnceGivenMultipleMatchesHTML()
 
         // Then
         assertThat(alertsRaised, hasSize(1));
-        assertEquals(alertsRaised.get(0).getEvidence(), "eval");
+        assertTrue(
+                alertsRaised.get(0).getEvidence().equals("bypassSecurityTrustHtml")
```

Comment:
I'd rather stick to a Set for scalability, I feel that it's a worthy tradeoff since the only downside over using a List is non-deterministic iteration which only affects testing. I addressed that in my latest commit so am thinking it should be fine?

Reply:
What do you mean by scalability?

Reply:
If the function list grows or if the payload gets very big.

Reply:
That shouldn't be a problem, the file/payloads will not have millions of entries.

Reply:
Gotcha, will switch to using a List then :)